docs/ledger/migrations/v5.0-migration.md

```diff
@@ -0,0 +1,105 @@
+# Migration Guide v5.0
+
+## API Changes
+
+### cosmos
+
+Regen Ledger v5.0 includes [Cosmos SDK 0.46](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.0).
+
+#### cosmos.groups
+`x/group` provides functionality to define on-chain groups of people that can execute arbitrary messages based on agreed upon rules.
+
+### regen.intertx
+
+Regen Ledger v5.0 introduces [regen.intertx.v1](https://buf.build/regen/regen-ledger/docs/main:regen.data.v1).
+
+### regen.ecocredit.marketplace
+
+Regen Ledger v5.0 includes non-breaking changes to [regen.marketplace.v1](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.marketplace.v1).
+
+- [v1.MsgAddAllowedDenom](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.marketplace.v1#regen.ecocredit.marketplace.v1.MsgAddAllowedDenom)
+
+- [v1.MsgAddAllowedDenomResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.marketplace.v1#regen.ecocredit.marketplace.v1.MsgAddAllowedDenomResponse)
+
+- [v1.MsgRemoveAllowedDenom](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.marketplace.v1#regen.ecocredit.marketplace.v1.MsgRemoveAllowedDenom)
+
+- [v1.MsgRemoveAllowedDenomResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.marketplace.v1#regen.ecocredit.marketplace.v1.MsgRemoveAllowedDenomResponse)
+
+#### Query Service
+
+  - no changes
+
+#### Types
+
+- [v1.AllowDenomProposal](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.marketplace.v1#regen.ecocredit.marketplace.v1.AllowDenomProposal)
+
+  - deprecated `AllowDenomProposal`
+
+#### Events
+
+- [v1.EventRemoveAllowedDenom](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.marketplace.v1#regen.ecocredit.marketplace.v1.EventRemoveAllowedDenom)
+
+### regen.ecocredit.v1
+
+#### Msg Service
+
+  - no changes
+
+#### Query Service
+
+- [BalancesByBatch](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.BalancesByBatch)
```

Comment:
This is the rpc method but we should list the query and query response. For example, this would require including:

- QueryBalancesByBatch
- QueryBalancesByBatchResponse

This will need to be updated for the others below as well.

Reply:
For clarification, we can remove the rpc methods from the migration guide (i.e. only include msg/query and response) but events and types are correct.